doc/source/tools/libraries/index.rst

```diff
@@ -1,7 +1,7 @@
 Tooling libraries
 =================
 
-.. grid:: 3
+.. grid:: 2
```

Comment:
Why this grid change¿?

Reply:
All pages are using 3 - it's uniformity. Otherwise, every time we add a new one we would have to change it

Reply:
So there was no row with one single element at the end. I agree with you, I'll revert